### ISSUE TITLE
Blinded UTXOs in Asset Import

### ISSUE THREAD

**Issue by cryptoquick** [bitmask-stack/bitmask-core#138]
I ran into an issue with asset_import:
https://github.com/diba-io/bitmask-core/blob/4b686330d79e748b759b572a61fd84215a259a0d/src/lib.rs#L316

In practice this results in a request like this:
```
{
  "asset": "rgb19jkjhpsjvlmktzhdax8dpmhwx59eafkqhhy3znswwf40l5e0ppuq6damnl",
  "utxos": [
    "266d2190920eb02e89c6a0425c2ce2fe2bd19f4660245023c97213eaef942c19:1",
    "txob1v4hymp5sd5j8rnn4sl0rz0se7w94rus8vhwmh72s6sy8kg7es9eqn3ml4u"
  ]
}
```

This results in this error:
```
Something went wrong: OutPoint not in <txid>:<vout> format
```

I'm not sure the best way to handle this. I'm hoping @crisdut will know more.

**Comment by oacdutra** [comment]
Hi @cryptoquick,

I'm sure why it is needed. This is added in this PR: https://github.com/diba-io/bitmask-core/pull/136

I think this is necessary to construct the asset allocation state, but does not work with blind UTXO information. 

We can change the parameter type from `Outpoint` to `BlindedOrNotOutpoint`.

**Comment by cryptoquick** [comment]
Maybe so, but they're two different kinds of data being mixed in, and so it causes an error:
https://github.com/diba-io/bitmask-core/pull/136/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R315

**Comment by oacdutra** [comment]
I think this method is incomplete.

Can you would like I open a PR to fix that?

**Comment by cryptoquick** [comment]
Please do, and maybe branch from @josediegorobles's changes in #137

**Comment by oacdutra** [comment]
@cryptoquick,

I think this issue can be closed, correct?

**Comment by cryptoquick** [comment]
Hmm... It's still a bug that exists in development, so I'll make sure this closes when #137 closes.
